weave-js/src/components/Form/Select.tsx

```diff
@@ -226,9 +225,10 @@ const getStyles = <
           ? `0 0 0 2px ${colorBorderOpen}`
           : `inset 0 0 0 1px ${colorBorderDefault}`,
         '&:hover': {
-          boxShadow: state.menuIsOpen
-            ? `0 0 0 2px ${colorBorderOpen}`
-            : `0 0 0 2px ${colorBorderHover}`,
+          boxShadow:
+            state.menuIsOpen || state.isFocused
+              ? `0 0 0 2px ${colorBorderOpen}`
+              : `0 0 0 2px ${colorBorderHover}`,
```

Comment:
Adding updated hover / focus colors for select component
![Screenshot 2024-11-04 at 2 11 17 PM](https://github.com/user-attachments/assets/095b1c80-5e45-4908-9d47-4c75d66c90ab)
![Screenshot 2024-11-04 at 2 11 23 PM](https://github.com/user-attachments/assets/1a751196-d4b5-4e27-ab8e-7a3865e81b6f)
Also fixes issue where the hover color would show when the item was already focused